src/Avalonia.Visuals/Rendering/IRenderLoopTask.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace Avalonia.Rendering
+{
+    public interface IRenderLoopTask
+    {
+        bool NeedsUpdate { get; }
+        void Update(long tickCount);
+        void Render();
+    }
+
+    public class MockRenderLoopTask : IRenderLoopTask
```

Comment:
Is this needed?

Reply:
I added this for testing. I'll remove it.